docs/data-sources/connectors.md

```diff
@@ -22,6 +29,12 @@ data "twingate_connectors" "all" {}
 ### Optional
 
 - `connectors` (Attributes List) List of Connectors (see [below for nested schema](#nestedatt--connectors))
```

Comment:
Is there a difference between

```
data "twingate_connectors" "all" {
  connectors = {
    name = "test"
  }
}  
``` 

and

```
data "twingate_connectors" "all" {
  name = "test"
}  
```

Do we still need the `connectors` attribute? If so should we just add the filters to it?